integration-tests/build.rs

```diff
@@ -0,0 +1,37 @@
+use std::{env, path::Path, process::Command};
+
+// make sure these are in sync with the values in tests/upgrade.rs
+const OLD_CONTRACT_VERSION: &str = "0.1.0";
```

Comment:
We should have clear instructions in the `README.md` on how to update this version.